core/templates/pages/oppia_footer_directive.html

```diff
@@ -20,7 +20,7 @@ <h4 translate="I18N_FOOTER_ABOUT_ALL_CAPS"></h4>
             </a>
           </li>
           <li>
-            <a href="/about#credits"
+            <a
                translate="I18N_FOOTER_CREDITS">
             </a>
```

Comment:
The link will still exist, right? Can you please update the "Proof that changes are correct" section of this PR?

Reply:
With these changes The Credits option is present in the footer but it is unclickable. Is it expected to remove the whole credits option from the footer?

Reply:
Why have a button/link which can't do anything? (It is somewhat the same as a broken link from the UX perspective.)

Reply:
> Also, can you please update the "Proof that changes are correct" section of this PR?

Is this^ resolved?

Reply:
done ptal.